dts/arm/raspberrypi/rpi_pico/rp2350.dtsi

```diff
@@ -435,4 +435,12 @@
 		compatible = "raspberrypi,pico-temp";
 		status = "disabled";
 	};
+
+	id_code: id_code@10000000 {
+		compatible = "zephyr,memory-region";
+		zephyr,memory-region = "IMAGE_DEF_FLASH";
+		zephyr,memory-region-flags = "r";
+		reg = <0x10000000 0x80>;
+		status = "okay";
```

Comment:
Please match the style here https://docs.kernel.org/devicetree/bindings/dts-coding-style.html#order-of-properties-in-device-node . 

It's not mandated by the Zephyr Project, but conscious effort was made to ensure this was the case elsewhere in this file.

Reply:
addressed.